_opam-files/META.lwt.template

```diff
@@ -0,0 +1,69 @@
+# JBUILDER_GEN
+
+version = "dev"
+description = "[compatibility] maps to lwt-core"
+requires = "lwt-core"
+
+package "unix" (
+  requires = "lwt-unix"
+  description = "[compatibility] maps to lwt-unix"
+  exists_if = "../lwt-unix/lwt_unix.cma"
+)
+
+package "log" (
+  requires = "lwt-log"
+  description = "[compatibility] maps to lwt-log"
+  exists_if = "../lwt-log/lwt_log.cma"
+)
+
+package "ppx" (
+  requires = "lwt-ppx"
+  description = "[compatibility] maps to lwt-ppx"
+  exists_if = "../lwt-ppx/ppx_lwt.cma"
+)
+
+package "preemptive" (
+  requires = "lwt-preemptive"
+  description = "[compatibility] maps to lwt-preemptive"
+  exists_if = "../lwt-preemptive/lwt_preemptive.cma"
+)
+
+package "simple-top" (
+  requires = "lwt-simple-top"
+  description = "[compatibility] maps to lwt-simple-top"
+  exists_if = "../lwt-simple-top/lwt_simple_top.cma"
+)
+
+package "glib" (
+  requires = "lwt-glib"
+  description = "[compatibility] maps to lwt-glib"
+  exists_if = "../lwt-glib/lwt_glib.cma"
+)
+
+package "ssl" (
+  requires = "lwt-ssl"
+  description = "[compatibility] maps to lwt-ssl"
+  exists_if = "../lwt-ssl/lwt_ssl.cma"
+)
+
+package "react" (
+  requires = "lwt-react"
+  description = "[compatibility] maps to lwt-react"
+  exists_if = "../lwt-react/lwt_react.cma"
+)
+
+package "syntax" (
+  requires = "camlp4 lwt-camlp4 lwt.camlp4.options"
+  exists_if = "../lwt-camlp4/lwt_camlp4.cma"
```

Comment:
About half way through the PR I was taught that splitting up the packages wasn't necessary, so I decided to keep the current package/sub-package organisation.

Whether splitting up the package is desirable, is a bigger conversation, and generally speaking I think a positive move, so instead of removing the various opam files I put them in this `_opam-files` directory.  With these files, and some grepping and editing of jbuild files, you can radically change the package organisation in 10 minutes which I think is an awesome jbuilder feature.

Specifically, this lwt.opam file was designed to model the current lwt package on top of a potential new set of packages.  The exists_if lets it expose these new packages *if installed*, which it otherwise doesn't know the status of.  There are issues with this scheme - while it helps compatibility, doing `opam remove lwt` doesn't actually uninstall lwt - just the package name mappings.  I also find the relative directory hijinks rather smelly.